pybossa/cache/project_stats.py

```diff
@@ -315,7 +316,10 @@ def stats_hours(project_id, period='2 week'):
     for row in results:
         max_hours = row.max
 
+    # If DISABLE_ANONYMOUS_ACCESS is True, deep copy (shallow copy also works) the data of 'all users' to avoid expensive query
+    # Also skip Anonymous users' hour stats and maximum stats
```

Comment:
1. Can deepcopy be removed if not needed; less code easy to maintain :). With this, import and comments be rephrased as below or simiar
```python
# with anonymous access disabled, auth users and aggr users counts would be same.
# reuse aggr user counts for auth users.  
```
2. Seems default theme changes not needed for this pr, hence can be excluded from it.

Looks good otherwise.

Reply:
I was thinking it was a good practice to copy dictionary using deep copy, although in our case here, the value is just a Number type, meaning the shallow copy and deep copy is the same for this case. But in the future, if we decide to use a list for values, the shallow copy here potentially will cause a bug. However, by reading through the code, this is unlikely. Also, even we change the value to a collection, we should use immutable collections like tuple.

Reply:
:+1: